source/Locales/ContinuousMap/FrameIsomorphism-Definition.lagda

```diff
@@ -0,0 +1,245 @@
+
+\begin{code}
+
+ is-homomorphic : (⟨ F ⟩ ≃ ⟨ G ⟩) → Ω (𝓤 ⊔ 𝓥 ⊔ 𝓤' ⊔ 𝓥' ⊔ 𝓦 ⁺)
+ is-homomorphic e = is-a-frame-homomorphism F G ⌜ e ⌝
+                  ∧ is-a-frame-homomorphism G F (inverse ⌜ e ⌝ (⌜⌝-is-equiv e))
+
+\end{code}
+
+The type of isomorphisms between `F` and `G` could alternatively be defined
+as the type of homomorphic equivalences.
+
+\begin{code}
+
+ Isomorphism₀ : 𝓤 ⊔ 𝓤' ⊔ 𝓥 ⊔ 𝓥' ⊔ 𝓦 ⁺  ̇
+ Isomorphism₀ = Σ e ꞉ ⟨ F ⟩ ≃ ⟨ G ⟩ , is-homomorphic e holds
+
+\end{code}
+
+These two notions of frame isomorphism are equivalent.
+
+\begin{code}
+
+ isomorphismᵣ-to-isomorphism₀ : Isomorphismᵣ → Isomorphism₀
+ isomorphismᵣ-to-isomorphism₀ iso = (s , † , ‡) , φ , ψ
+  where
+   open Isomorphismᵣ iso
+
+   † : has-section s
+   † = r , forward-cancels-backward
+
+   ‡ : is-section s
+   ‡ = r , backward-cancels-forward
+
+   φ : is-a-frame-homomorphism F G s holds
+   φ = fun-is-a-frame-homomorphism F G forward
+
+   ψ : is-a-frame-homomorphism G F r holds
+   ψ = fun-is-a-frame-homomorphism G F backward
+
+ isomorphism₀-to-isomorphismᵣ : Isomorphism₀ → Isomorphismᵣ
+ isomorphism₀-to-isomorphismᵣ (e , φ , ψ)  =
+  record
+   { forward                  = ⌜ e ⌝ , φ
+   ; backward                 = inverse ⌜ e ⌝ (⌜⌝-is-equiv e) , ψ
+   ; backward-cancels-forward = inverses-are-retractions ⌜ e ⌝ (⌜⌝-is-equiv e)
+   ; forward-cancels-backward = inverses-are-sections ⌜ e ⌝ (⌜⌝-is-equiv e)
+   }
+
+ isomorphism-to-isomorphism₀ : Isomorphism → Isomorphism₀
+ isomorphism-to-isomorphism₀ =
+  isomorphismᵣ-to-isomorphism₀ ∘ isomorphism-to-isomorphismᵣ
+
+ isomorphism₀-to-isomorphism : Isomorphism₀ → Isomorphism
+ isomorphism₀-to-isomorphism =
+  isomorphismᵣ-to-isomorphism ∘ isomorphism₀-to-isomorphismᵣ
+
+ isomorphism-equivalent-to-isomorphism₀ : Isomorphism ≃ Isomorphism₀
+ isomorphism-equivalent-to-isomorphism₀ = isomorphism-to-isomorphism₀
+                                        , (isomorphism₀-to-isomorphism , †)
+                                        , (isomorphism₀-to-isomorphism , ‡)
+  where
+   † : isomorphism-to-isomorphism₀ ∘ isomorphism₀-to-isomorphism ∼ id
+   † (h , _) =
+    to-subtype-＝
+     (holds-is-prop ∘ is-homomorphic)
+     (to-subtype-＝ (being-equiv-is-prop (λ 𝓤 𝓥 → fe {𝓤} {𝓥})) refl)
+
+   ‡ : isomorphism₀-to-isomorphism ∘ isomorphism-to-isomorphism₀ ∼ id
+   ‡ iso = to-subtype-＝ (holds-is-prop ∘ is-isomorphism) refl
+
+ isomorphismᵣ-equivalent-to-isomorphism₀ : Isomorphismᵣ ≃ Isomorphism₀
+ isomorphismᵣ-equivalent-to-isomorphism₀ =
+  Isomorphismᵣ   ≃⟨ Ⅰ ⟩
+  Isomorphism    ≃⟨ Ⅱ ⟩
+  Isomorphism₀   ■
+   where
+    Ⅰ = ≃-sym isomorphism-equiv-to-isomorphismᵣ
+    Ⅱ = isomorphism-equivalent-to-isomorphism₀
+
+\end{code}
+
+Declare syntax for frame isomorphisms.
+
+\begin{code}
+
+Isomorphismᵣ-Syntax : Frame 𝓤 𝓥 𝓦 → Frame 𝓤' 𝓥' 𝓦 → 𝓤 ⊔ 𝓤' ⊔ 𝓥 ⊔ 𝓥' ⊔ 𝓦 ⁺  ̇
+Isomorphismᵣ-Syntax = FrameIsomorphisms.Isomorphismᵣ
+
+infix 0 Isomorphismᵣ-Syntax
+syntax Isomorphismᵣ-Syntax F G = F ≅f≅ G
```

Comment:
Wouldn't `≅Fr≅` or even `≅⟨Fr⟩`  be better? I leave it up to you.

Reply:
Yes I think that might actually be better. I think it is closer to conventions that I’ve seen in other parts of TypeTopology.